modules/paper/src/main/kotlin/io/github/lmliam/kotventure/paper/dialog/input/TextInputBuilder.kt

```
@@ -0,0 +1,71 @@
+internal class TextInputBuilder(
+    private val key: String,
+) : TextInputScope {
+    private var label: Component? by once()
+    private var labelVisible: Boolean? by once()
+    private var width: Int? by once().inRange(1..1024)
+    private var default: String? by once()
+    private var maxLength: Int? by once().positive()
+    private var multilineOptions: TextDialogInput.MultilineOptions? by once()
+
+    override fun label(init: LabelScope.() -> Unit) {
+        lateinit var labelBuilder: LabelBuilder
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)